var/spack/repos/builtin/packages/daos/package.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Daos(SConsPackage):
+    """The Distributed Asynchronous Object Storage (DAOS) is an open-source
+    software-defined object store designed from the ground up for massively
+    distributed Non Volatile Memory (NVM)."""
+
+    homepage = "https://github.com/daos-stack/daos"
+    git = "https://github.com/daos-stack/daos.git"
+    maintainers("hyoklee")
+
+    version("master", branch="master", submodules=True)
+    version("2.2.0", tag="v2.2.0", submodules=True)
+    variant(
+        "debug", default=False, description="Enable debugging info and strict compile warnings"
+    )
+
+    depends_on("argobots@1.1:")
+    depends_on("boost@develop+python", type="build")
```

Comment:
I don't think we need develop here ? whatever version should be fine ?

Reply:
Why was this conversation resolved, while `boost@develop` is still here? Am I missing some comments?

Reply:
Also, can you clarify why `boost` is a `build` dependency?